bindsnet/datasets/alov300.py

```diff
@@ -0,0 +1,318 @@
+from __future__ import print_function, division
+
+import os
+import numpy as np
+import torch
+import shutil
+import zipfile
+import sys
+import time
+import warnings
+import cv2
+import random
+import bindsnet.datasets.preprocess
+
+
+from PIL import Image
+from glob import glob
+from tqdm import tqdm
+from collections import defaultdict
+from typing import Optional, Tuple, List, Iterable
+from urllib.request import urlretrieve
+from torchvision import transforms
+import xml.etree.ElementTree as ET
```

Comment:
Is this `import` used anywhere? I tried searching but couldn't find where it's used.

Glancing through, a lot of these `import`s don't seem to be used? eg `defaultdict`, the typings, `tqdm`, etc

Reply:
Yeah these were used before refactoring. I may still use TQDM but I'll remove it for now